react-native-sdk/update_dependencies.js

```diff
@@ -21,6 +24,19 @@ function updateDependencies() {
             packageJSON.dependencies[key] = RNSDKpackageJSON.peerDependencies[key];
             updated = true;
         }
+
+        if (semver.satisfies(RNSDKpackageJSON.peerDependencies[key], `=${packageJSON.dependencies[key]}`)) {
+            return;
```

Comment:
You want to keep checking other packages no? So `continue` then.

Reply:
Why continue? We are comparing all of our peer dependencies with the dependencies that have the same property. If they all have the same version, that's what we expect and we stop.